docs/tutorials/detect.rst

```diff
@@ -0,0 +1,246 @@
+                             "S": [2, 9.9, 2]}
+
+High- and low-cut frequencies can be defined for both P- and S-waves separately and are defined in Hertz. Typically S-waves have lower frequency content than P-waves and the horizontal components may have a different seismic-noise frequency content to the vertical component.
+
+For volcano-tectonic or microseismicity recorded across a local network, good starting values are 2 Hz for the low-cut corner and 10 - 16 Hz for the high-cut corner. These values remove most of the seismic noise associated with the oceanic microseism (periods > 7 s) and reduce high-frequency anthropogenic seismic noise.
+
+.. note:: QuakeMigrate uses a Butterworth filter with customisable high- and low-corners. The filter is applied both forwards and backwards to remove any phase shifts. As such, the effective order is double the user-defined order.
+
+.. note:: Remember to check the Nyquist frequency of your data. An exception will be thrown if you try to filter at frequencies greater than the Nyquist.
+
+Window lengths
+##############
+In combination with your filter choice, the choice of window length is the most important parameter in producing high-quality results. As when specifying the filter parameters, you can choose to specify different window parameters for P- and S-phases to account for differences in their frequency content and/or noise value. 
+
+::
+
+    # [length of short window (s), length of long window (s)]
+    onset.sta_lta_windows = {"P": [0.2, 1.5],
+                             "S": [0.2, 1.5]}
+
+A good place to start is to choose a short window length equal to 2-3 times the dominant period of the signal you are hoping to capture. The long window values are then much longer than the short window. Typical values are 5-10 times the length of the short window. 
+
+When choosing your parameters, you should experiment with different values using your data before running a `detect` run. A good way to do this is to use a combination of the :class:`Archive` and :class:`STALTAOnset` classes to grab sections of data from your archive and apply different filter and STALTA parameters to it. The data can be manipulated and displayed using `matplotlib <https://matplotlib.org/>`_ as in the example below.
+
+::
+
+    import matplotlib.pyplot as plt
+    from quakemigrate.signal.onsets import STALTAOnset
+    from quakemigrate.io import Archive, read_stations
+
+    # define an archive object
+    archive_path = "/path/to/archived/data"
+    station_file = "/path/to/station_file"
+    stations = read_stations(station_file)
+    archive = Archive(archive_path=archive_path, stations=stations,
+                      archive_format="YEAR/JD/STATION")
+                    
+    # Read a snippet of data (ideally around a known event)
+    starttime = "2018-001T10:00:00.0"
+    endtime = "2018-001T10:05:00.0"
+    data = archive.read_waveform_data(starttime, endtime, sampling_rate)
```

Comment:
In here, I´m having an error: 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/nomin/anaconda3/envs/quakemigrate/lib/python3.9/site-packages/quakemigrate/io/data.py", line 278, in read_waveform_data
    files = self._load_from_path(starttime - pre_pad, endtime + post_pad)
TypeError: unsupported operand type(s) for -: 'str' and 'float'